samples/streaming_sample/sample_stream.go

```diff
@@ -0,0 +1,215 @@
+func handleAddition(deltas []openapi.ModelNodeIdentifier) {
+	log.Info().Msgf("Addition Delta received: %v", deltas)
+
+	stdDeltas := make([]openapi.ModelNodeIdentifier, 0)
+	complianceDeltas := make([]openapi.ModelNodeIdentifier, 0)
+
+	for _, entity := range deltas {
+		switch strings.ToLower(entity.NodeType) {
+		case "container", "containerimage":
+			stdDeltas = append(stdDeltas, entity)
+		case "node":
+			stdDeltas = append(stdDeltas, entity)
+			complianceDeltas = append(complianceDeltas, entity)
+		}
+	}
+	if len(stdDeltas) > 0 {
+		startSecretScan(stdDeltas)
+		startMalwareScan(stdDeltas)
+		startVulnerabilityScan(stdDeltas)
+	}
+
+	if len(complianceDeltas) > 0 {
+		startComplianceScan(complianceDeltas)
+	}
+}
+
+func handleDeletion(deltas []openapi.ModelNodeIdentifier) {
+	log.Info().Msgf("Deletion Deltas received: %v", deltas)
+	//Add the code to handle the deletion deltas in this function
+}
+
+func startSecretScan(deltas []openapi.ModelNodeIdentifier) {
+	secretScanApi := sampleApiClient.Client().SecretScanAPI
+
+	secretScanReq := openapi.ModelSecretScanTriggerReq{}
+	secretScanReq.Filters = *(openapi.NewModelScanFilterWithDefaults())
+	secretScanReq.NodeIds = deltas
+
+	req := secretScanApi.StartSecretScan(context.Background())
+	req = req.ModelSecretScanTriggerReq(secretScanReq)
+	res, _, err := secretScanApi.StartSecretScanExecute(req)
+	if err != nil {
+		log.Info().Msgf("Error in starting secret scan: %v", err)
+		return
+	}
+
+	respStr, err := json.Marshal(res)
+	if err != nil {
+		log.Info().Msgf("Failed to json marshal response, error:%v", err)
+		return
+	}
+	log.Info().Msgf("Successfully started scan for deltas: %v", deltas)
+	log.Info().Msgf("StartSecretScan Response: %s", respStr)
+}
+
+func startMalwareScan(deltas []openapi.ModelNodeIdentifier) {
+	malwareScanApi := sampleApiClient.Client().MalwareScanAPI
+	malwareScanReq := openapi.ModelMalwareScanTriggerReq{}
+
+	malwareScanReq.Filters = *(openapi.NewModelScanFilterWithDefaults())
+	malwareScanReq.NodeIds = deltas
+	req := malwareScanApi.StartMalwareScan(context.Background())
+	req = req.ModelMalwareScanTriggerReq(malwareScanReq)
+	res, _, err := malwareScanApi.StartMalwareScanExecute(req)
+	if err != nil {
+		log.Info().Msgf("Error in starting malware scan: %v", err)
+		return
+	}
+
+	respStr, err := json.Marshal(res)
+	if err != nil {
+		log.Info().Msgf("Failed to json marshal response, error:%v", err)
+		return
+	}
+
+	log.Info().Msgf("Successfully started Malware scan for deltas: %v", deltas)
+	log.Info().Msgf("StartMalwareScan Response: %s", respStr)
+
+}
+
+func startVulnerabilityScan(deltas []openapi.ModelNodeIdentifier) {
+	vulnerabilityScanApi := sampleApiClient.Client().VulnerabilityAPI
+	vulnerabilityScanReq := openapi.ModelVulnerabilityScanTriggerReq{}
+
+	vulnerabilityScanReq.Filters = *(openapi.NewModelScanFilterWithDefaults())
+	vulnerabilityScanReq.NodeIds = deltas
+	scanConfig := make([]openapi.ModelVulnerabilityScanConfigLanguage, 0)
```

Comment:
This can be simplified and added in global var

Reply:
I took the cue from how we create clients in other sdk. 
e.g.: Creating a TLS config before initiating http client:
https://github.com/deepfence/golang_deepfence_sdk/blob/2fc0ef2007cd4c68efc5aeabeb125df733ebc940/utils/http/client.go#L147